docker-storage-setup.sh

```diff
@@ -395,6 +395,21 @@ systemd_escaped_filename () {
   echo $filename
 }
 
+
+#
+# In the past we created a systemd mount target file, we no longer
+# use it, but if one pre-existed we still need to handle it.
+#
+remove_systemd_mount_target () {
+  local mp=$1
+  local filename=$(systemd_escaped_filename $mp)
+  if [ -f /etc/systemd/system/$filename ];then
+    systemctl disable $filename >/dev/null 2>&1
```

Comment:
lets call `systemctl stop $filename` after disabling it. That unmounts any mount done by systemd.